sites/en/frontend/deploying_to_github_again.step

```diff
@@ -0,0 +1,11 @@
+h2 do
+  span "If you haven't deployed your app to GitHub yet, start at "
+  a 'Deploying to GitHub', href: 'deploying_to_github_pages'
+  span "."
+end
+
+important "If you have any problems with these steps, ask a volunteer for help. If you don't know if you have Git installed or have a GitHub account, it's okay to do these steps later instead."
+
+insert '_deploying_to_github'
```

Comment:
This page is inserting both `_deploying_to_github` and `_deploying_to_github_again`, so the `again` page is actually longer than the original one, which I don't think is right :)

I think the content of the `_deploying_to_github` and `_deploying_to_github_again` partials could just be inlined up to `deploying_to_github.step` and `deploying_to_github_again.step`, because the partials are never reused in any other situations.